src/core/control/tools/ImageSizeSelection.cpp

```diff
@@ -0,0 +1,42 @@
+#include "ImageSizeSelection.h"
+
+#include "util/Rectangle.h"  // for Rectangle
+
+
+using xoj::util::Rectangle;
```

Comment:
nitpick: I'd avoid this `using`: it only saves you 1 occurrence of `xoj::util::` and impedes readability a little bit.